R/check-data-format.R

```diff
@@ -27,19 +29,21 @@
 #' check_data_format(
 #'   outing = outing,
 #'   template = demo_template_fish_exploit,
-#'   complete = FALSE
+#'   complete = FALSE,
+#'   join = FALSE
 #' )
 #'
 #' check_data_format(
-#'   site = site
+#'   site = site,
 #'   outing = outing,
 #'   capture = capture,
 #'   recapture = recapture,
 #'   template = demo_template_fish_exploit,
-#'   complete = TRUE
+#'   complete = TRUE,
+#'   join = TRUE
 #' )
 #' }
-check_data_format <- function(..., template, complete = FALSE) {
+check_data_format <- function(..., template, complete = FALSE, join = FALSE) {
   chk::chk_used(...)
   chk::chk_list(template)
   chk::chk_lgl(complete)
```

Comment:
There should be a check added for the new join argument

Reply:
Added!

Reply:
Actual please update them to `cha::cha_flag()`